### ISSUE TITLE
All optimizations aren't compatible with 3d layers

### ISSUE THREAD

**Issue by hyperfraise** [NVIDIA/TensorRT#776]
Hi. I'm very interested in being able to use the full range of optimizations on 3d convolution BatchNorms and Pooling layers. Do you know when it might be ready ?

Some ressource indicating what is missing :  https://docs.nvidia.com/deeplearning/tensorrt/support-matrix/index.html
My speedup on 3d networks are pretty low for now compared to FP16 inference in pytorch.

**Comment by hyperfraise** [comment]
Still no full support for 3d operations. No quantization for example. Is there plans to improve this ?
